rskj-core/src/main/resources/config/rsk-sample.conf

```diff
@@ -188,17 +188,16 @@ sync {
 }
 
 rpc {
-    enabled = true
-    # Interface to bind rpc
-    # Make sure you are using the correct bind. Default value: localhost
-    address = localhost
-    port = 4444
-
-    # A value greater than zero sets the socket value in milliseconds. Node attempts to gently close all TCP/IP connections with proper half close semantics,
```

Comment:
Should this be moved into `rpc.providers.web.http`? You removed the configuration example but it's still being used to instantiate `Web3HttpServer`

Reply:
Please add a sample of the websocket config.